project_status/__manifest__.py

```diff
@@ -1,28 +1,22 @@
 {
-    'name': "Project Status",
-
-    'summary': """
+    "name": "Project Status",
+    "summary": """
         Project Status""",
-
-    'author': "Patrick Wilson, Odoo Community Association (OCA)",
-    'website': "https://github.com/OCA/project",
-
-    'category': 'Project Management',
-    'version': '12.0.1.1.1',
-    'license': 'AGPL-3',
-
+    "author": "Patrick Wilson, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/project",
+    "category": "Project Management",
+    "version": "13.0.1.1.1",
```

Comment:
Nope, it's not, don't mix reformatting and version change. On top of that, version is not correct

Reply:
I remembered that when I run pre-commit command, it failed. So I changed version from 12 to 13 and it passed.

Reply:
Will ocabot merge command will take care it?

Reply:
No it won't. The [MIG] commit should contain change 12.0.1.1.1 => 13.0.1.0.0

Reply:
And the reformat commit should not have a version change at all

Reply:
Let me try it one more time.

Reply:
Comment addressed!